### ISSUE TITLE
[mingw64] libtiff build failure

### ISSUE THREAD

**Issue by crhepner** [m-ab-s/media-autobuild_suite#1971]
The libtiff-git repo has failed for me for the past couple of weeks, with the same error message each time:

``` bash
  Running git update for libtiff...
┌ libtiff git  ........................................ [Recently updated]
233.patch
        Patch could not be applied with `git am`. Continuing without patching.
├ Running cmake...
├ Running build...
Likely error (tail of the failed operation logfile):
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1e1a): undefined reference to `__imp_glVertexPointer'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1e2a): undefined reference to `__imp_glNormalPointer'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1e4d): undefined reference to `__imp_glTexCoordPointer'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1e69): undefined reference to `__imp_glDrawElements'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1e9e): undefined reference to `__imp_glDisableClientState'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1f93): undefined reference to `__imp_glDrawElements'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_geometry.c.obj):fg_geometry.c:(.text+0x1fac): undefined reference to `__imp_glDrawArrays'
C:/media-autobuild_suite-master/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/media-autobuild_suite-master/local64/lib/libglut.a(fg_ext_mswin.c.obj):fg_ext_mswin.c:(.text+0x123): undefined reference to `__imp_wglGetProcAddress'
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
build failed. Check C:/media-autobuild_suite-master/build/libtiff-git/build-64bit/ab-suite.build.log
This is required for other packages, so this script will exit.
  Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/-aEp.zip
Copy and paste [logs.zip](https://0x0.st/-aEp.zip) in the GitHub issue.
```

Here are the log files: [logs.zip](https://0x0.st/-aEp.zip)

**Comment by 1480c1** [comment]
I updated the patch, but I'm curious to know what that patch failed for you since it wasn't marked as having a conflict

**Comment by crhepner** [comment]
Thanks!  I don't know if this is helpful or not, but before I replaced the old patch with your updated one, I tried applying it manually:

```bash
cd ./build/libtiff-git
git am 233.patch
```

That seemed to work and did not produce any error messages.  To test the updated patch, I deleted the libtiff-git folder and ran media-autobuild_suite.bat again.  It successfully applied the patch and compiled libtiff.
